ibis/sql/sqlite/compiler.py

```diff
@@ -144,6 +144,39 @@ def _extract_epoch_seconds(t, expr):
     return sa.cast(sa_expr, sa.BigInteger)
 
 
+def _extract_week_of_year(t, expr):
```

Comment:
Not sure if I'm missing something, but in [this comment](https://github.com/ibis-project/ibis/pull/2177#discussion_r446406556) Jeff said that if the db doesn't support the operation, let's simply not have it. Shouldn't we be removing this function then?

Reply:
I can remove it too. I will push the changes in a bit.